pkg/diagnose/connectivity/endpoint_info.go

```diff
@@ -30,23 +32,28 @@ type endpointInfo struct {
 }
 
 var (
-	// Each added/modified endpointInfo should be tested on all sites.
-
 	emptyPayload    = []byte("{}")
 	checkRunPayload = []byte("{\"check\": \"test\", \"status\": 0}")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why not move those in `getEndpointsInfo` too ?

Reply:
yes I will do it asap